From d4f3182a98f6f0c8477ba6840e41f01fa9a32020 Mon Sep 17 00:00:00 2001
From: Lu Yahan <yahan@iscas.ac.cn>
Date: Wed, 25 Aug 2021 09:55:50 +0800
Subject: [PATCH] [riscv64] riscv64 do not support Simd instructions

Change-Id: Idcb4f6263ff20866c166e44c33de1a37c4b6b896
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/3115743
Auto-Submit: Yahan Lu <yahan@iscas.ac.cn>
Commit-Queue: Jakob Gruber <jgruber@chromium.org>
Reviewed-by: Jakob Gruber <jgruber@chromium.org>
Cr-Commit-Position: refs/heads/main@{#76477}
---
 tools/testrunner/base_runner.py | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/tools/testrunner/base_runner.py b/tools/testrunner/base_runner.py
index 93b4fc4d044..19165b34360 100644
--- a/tools/testrunner/base_runner.py
+++ b/tools/testrunner/base_runner.py
@@ -688,6 +688,10 @@ class BaseTestRunner(object):
        utils.GuessPowerProcessorVersion() < 9:
        no_simd_hardware = True
 
+    # riscv64 do not support Simd instructions
+    if self.build_config.arch == 'riscv64':
+       no_simd_hardware = True
+
     return {
       "arch": self.build_config.arch,
       "asan": self.build_config.asan,
-- 
2.35.1

